lib/dhis2/api/event.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+module Dhis2
+  module Api
+    class Event < Base
+      class << self
+        def create(client, tuples)
+          begin
+            body = { events: tuples }
```

Comment:
Any reason this could not be in Base? Looks like only the symbol would change and it can be generated from the class name